tests/data/episode_podcast_chapters.json

```diff
@@ -0,0 +1,23 @@
+{
+    "title": "Podcast",
+    "episodes": [
+        {
+            "title": "Episode",
+            "description": "",
+            "published": 0,
+            "guid": "http://example.org/example.mp3",
+            "link": "",
+            "total_time": 0,
+            "payment_url": null,
+            "type": "trailer",
+            "enclosures": [
+                {
+                    "file_size": -1,
+                    "url": "http://example.org/example.mp3",
+                    "mime_type": "application/octet-stream"
+                }
+            ],
+            "chapters_json_url": "https://example.com/1/chapters.json"
+        }
+    ]
+}
```

Comment:
Please set your editor to add a newline at end of file. [Why?](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)